### PR TITLE
UDPBroadcastrelay UI fix for 24.7

### PR DESCRIPTION
**Pull request by marjohn56**
The UI was causing errors when running under 24.7, this issue was not present in 24.1 and earlier versions. Not sure what's causing it but removing the following:
 ```
if ($this->request->isPost()) {
            $this->sessionClose();
        }
```
fixes the issue.

### CODE REVIEW

File: net/udpbroadcastrelay/src/opnsense/mvc/app/controllers/OPNsense/UDPBroadcastRelay/Api/ServiceController.php [hunk 1]
```diff
@@ -44,10 +43,6 @@ class ServiceController extends ApiMutableModelControllerBase
 
     public function statusAction($uuid)
     {
-        $result = array("result" => "failed", "function" => "status");
```
Comment (fichtner) [inline net/udpbroadcastrelay/src/opnsense/mvc/app/controllers/OPNsense/UDPBroadcastRelay/Api/ServiceController.php:47]: you need to keep this line

File: net/udpbroadcastrelay/src/opnsense/mvc/app/controllers/OPNsense/UDPBroadcastRelay/Api/ServiceController.php [hunk 2]
```diff
@@ -126,10 +121,6 @@ public function restartAction($uuid)
      */
     public function configAction()
     {
-        $result = array("result" => "failed", "function" => "config");
```
Comment (fichtner) [inline net/udpbroadcastrelay/src/opnsense/mvc/app/controllers/OPNsense/UDPBroadcastRelay/Api/ServiceController.php:129]: keep this line

File: net/udpbroadcastrelay/src/opnsense/mvc/app/controllers/OPNsense/UDPBroadcastRelay/Api/ServiceController.php [hunk 3]
```diff
@@ -31,7 +31,6 @@
 use OPNsense\Base\ApiMutableModelControllerBase;
 use OPNsense\UDPBroadcastRelay\UDPBroadcastRelay;
 use OPNsense\Core\Backend;
-
```
Comment (fichtner) [inline net/udpbroadcastrelay/src/opnsense/mvc/app/controllers/OPNsense/UDPBroadcastRelay/Api/ServiceController.php:34]: if you can bring back this spacer line also ;)

Comment (marjohn56) [reply net/udpbroadcastrelay/src/opnsense/mvc/app/controllers/OPNsense/UDPBroadcastRelay/Api/ServiceController.php:34]: Sorry, took out more than I needed to. 
I have now retired and moved out of London and I am busier than ever,  that's my excuse and I'm sticking to it. :)

Comment (fichtner) [reply net/udpbroadcastrelay/src/opnsense/mvc/app/controllers/OPNsense/UDPBroadcastRelay/Api/ServiceController.php:34]: Living the good life I see. Glad to hear you're doing good! :)

**Comment by fichtner** [comment]
Merged manually via https://github.com/opnsense/plugins/commit/7a533a171bd24d92f465fbadd238223495e5e4aa ;)

**Pull request closed without merging**